server/helpers/utils.ts

```diff
@@ -80,14 +87,29 @@ function getUUIDFromFilename (filename: string) {
   return result[0]
 }
 
+let tmpPath
+async function getTmpPath (filename?: string) {
+  if (!tmpPath) {
+    try {
+      tmpPath = join(CONFIG.STORAGE.TMP_DIR, 'resumable-uploads')
```

Comment:
Create the directory in the installer function so you don't need an async function here: https://github.com/Chocobozzz/PeerTube/blob/develop/server/initializers/installer.ts#L80

And please use a better name than `getTmpPath`